modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -191,7 +191,7 @@
 
 			&:hover {
 				color: var( --n-tabs-title-color-hover );
-				background-color: var( --n-tabs-title-background-color-hover );
+				background: var( --n-tabs-title-background-color-hover );
```

Comment:
Shouldn't we remove these and move them to the new css selector with the higher specificity?

Reply:
I think that the CSS property should be the same in both cases.
In the control, we are targeting the background CSS property in the PHP.
If my explanation is not clear I will elaborate on the explanation.

Reply:
Can't we remove this and move it? Don't we have this in line 241 already?